server/src/test/java/org/apache/iotdb/db/metadata/MManagerBasicTest.java

```diff
@@ -957,6 +962,46 @@ public void testTemplateInnerTree() {
     }
   }
 
+  @Test
+  public void testCreateSchemaTemplateSerializationAdaptation() throws IOException {
+    CreateTemplatePlan plan = getCreateTemplatePlan();
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    DataOutputStream dos = new DataOutputStream(baos);
+    plan.formerSerialize(dos);
+    byte[] byteArray = baos.toByteArray();
+    ByteBuffer buffer = ByteBuffer.wrap(byteArray);
+
+    assertEquals(PhysicalPlan.PhysicalPlanType.CREATE_TEMPLATE.ordinal(), buffer.get());
+
+    CreateTemplatePlan deserializedPlan = new CreateTemplatePlan();
+    deserializedPlan.deserialize(buffer);
+
+    assertEquals(plan.getCompressors().size(), deserializedPlan.getCompressors().size());
+    assertEquals(
+        plan.getMeasurements().get(0).get(0), deserializedPlan.getMeasurements().get(0).get(0));
+    assertEquals(plan.getDataTypes().size(), deserializedPlan.getDataTypes().size());
+  }
```

Comment:
Maybe move to MLogUpgradeTest?

Reply:
it has been moved to MLogUpgradeTest now